pkg/cmd/server/kubernetes/master.go

```diff
@@ -481,3 +481,34 @@ func (c *MasterConfig) createSchedulerConfig() (*scheduler.Config, error) {
 	// if the config file isn't provided, use the default provider
 	return configFactory.CreateFromProvider(factory.DefaultProvider)
 }
+
+type noAutoApproval struct{}
+
+func (noAutoApproval) AutoApprove(csr *certificates.CertificateSigningRequest) (*certificates.CertificateSigningRequest, error) {
+	return csr, nil
+}
+
+func (c *MasterConfig) RunCertificateSigningController(clientset *kclientset.Clientset) {
```

Comment:
I don't like embedding this in our main controller process.  Can we kick it out of process and tell people to run such things in a separate pod in a separate namespace?